CHANGELOG

```diff
@@ -1,3 +1,31 @@
+Release 7 Adrianople (2017-12-15)
+----------------------------
+Common: bump OracleJDK to 8 in .travis.yml (close #281)
```

Comment:
We don't include "close " in the CHANGELOG

Reply:
Ah! Sorry, thanks for reminding!